.github/dependabot.yaml

```diff
@@ -5,12 +5,22 @@ updates:
       include: scope
       prefix: chore
     directory: /
-    ignore:
-      - dependency-name: '@types/node'
-        versions:
-          - '>= 13'
+    open-pull-requests-limit: 99
+    package-ecosystem: github-actions
+    reviewers:
+      - 'ridedott/platform'
+    schedule:
+      interval: daily
+  - allow:
+      - dependency-type: direct
```

Comment:
is this not already present on line 2-3 ?

Reply:
This is a separate entry in an array, a separate configuration object.

Reply:
Ahh yeah, i was unsure. Indenting hard to read in these diffs 🙈